blog/html_files/staffing-case-study.html

```diff
@@ -0,0 +1,75 @@
+  <p style="text-align:justify;">
+    <h2>Problem</h2>
+  </p>
+  <p style="text-align:justify;">In organizations that grow as quickly as our client’s, new employees entering the company into new roles, or roles that have been empty for a while, find that their swim lanes take time to be defined (or re-defined).With new roles, it can be difficult to define responsibilities properly until someone is actually sitting in the position, ramped up, and doing the work. In the case of existing roles that sit empty, work that should be completed either falls to the wayside, or has to be covered by the remaining members of the team, causing a strain on resourcing. With lead-times for hiring technical talent being longer than the average, this strain represents a challenge to a team that aims to excel in a high growth environment like the one we have here.&nbsp;</p>
+  <p style="text-align:justify;">
+  <h2>Solution</h2>
+  </p>
+  <p style="text-align:justify;">Snowpack provided analytics support to the client on an hourly basis. The contractor was assigned to fill in the role and contribute to the team until such time a replacement was hired.In this case, the nature of the work was BI focused–examples of the type of work done by the contractor are:
+  </p>
+  <ol>
+  <p style="text-align:justify;"> 1. Contributing to large scale projects, such as a redesign of financial data models.
+  </p>  
+  <p style="text-align:justify;"> 2. Driving metric development and weekly executive reporting for key strategic initiatives.
+  </p>
+  <p style="text-align:justify;"> 3. Conducting ad-hoc analysis on the user journey and supporting other analysts on the team in their analyses.
+  </p>
+  </ol>
+  <p style="text-align:justify;">The contractor filled in for a period of 7 months starting in August 2023, and was available day-to-day to cover additional responsibilities that came up during high demand periods.</p>
+  <p style="text-align:justify;">
+    <h2>Advantages of filling a temporary gap in staffing with a contractor</h2>
+  </p>
+  <ol>
+  <p style="text-align:justify;">
+    <strong>1. Minimal ramping time</strong>
+  </ol>
+  <p style="text-align:justify;">When projects are clearly scoped and defined, contractors can ramp much more quickly due to the narrower nature of project-based work vs. management of an entire analytics discipline or practice.&nbsp;</p>
+  <ol>
+  <p style="text-align:justify;">
+    <strong>2. Autonomy</strong>
+  </ol>
+  <p style="text-align:justify;">The expectation is not to handhold, but to provide the information the contractor needs to succeed. This translates to not having to explain key foundational concepts,&nbsp;</p>
```

Comment:
```suggestion
  <p style="text-align:justify;">The expectation is not to handhold, but to provide the information the contractor needs to succeed. This translates to not having to explain key foundational concepts.&nbsp;</p>
```

did you want a period here or did text get truncated?